en/messages.json

```diff
@@ -594,5 +594,79 @@
     "popupHolidayPromo2023CTA": {
         "message": "Get 1 year of Relay Premium",
         "description": "DO NOT TRANSLATE \"Relay Premium\"."
+    },
+    "popupPhoneMasksMetaForwardingDescription": {
+        "message": "Relay is currently forwarding all phone calls and text messages to your true phone number.",
+        "description": "DO NOT TRANSLATE \"Relay\"."
+    },
+    "popupPhoneMasksMetaBlockingDescription": {
+        "message": "Relay is blocking all phone calls and text messages—you will not receive anything from your phone number mask.",
+        "description": "DO NOT TRANSLATE \"Relay\"."
+    },
+    "popupPhoneMasksMetaForwardTo": {
+        "message": "Forward to:"
+    },
+    "popupPhoneMasksMetaRegistered": {
+        "message": "Registered:"
```

Comment:
What does "Registered" means in this context? Is there a screenshot? Once clarified, this string needs a comment (description) to clarify it.

Reply:
Added descriptions. Here is some context, see "registered" near the bottom.
![302500456-363be5ca-eebd-442a-9b01-74d2d67dab69](https://github.com/mozilla-l10n/fx-private-relay-add-on-l10n/assets/59676643/9118571d-ca3e-47a2-a640-f8dc11d9dfa3)

Reply:
Thanks, this is helpful. The image say Emails/Phones, I assume that changed to Email/Phone now, given the strings?

I have a feeling you're going to have issues with a 2-column layout, some languages will have longer translations, some potentially even single words that are longer than the available space. Was the layout tested with a test much longer, e.g. twice the length of English??

Reply:
Correct, that was an older screenshot, it is "Email" and "Phone" now.

Great catch, did some testing now and it did break the layout. Fixed it [here](https://github.com/mozilla/fx-private-relay-add-on/pull/553/commits/cc8836ab914ef25bd336e475394d709c0f672156), it now wraps the text incase translated strings are long.